sdk/provisioning/README.md

```diff
@@ -0,0 +1,222 @@
+# Azure Provisioning libraries for .NET
+
+The Azure Provisioning libraries are a cloud development kit that allow you to declaratively specify your Azure infrastructure natively in dotnet.
+Azure.Provisioning contains the core functionality for the Azure Provisioning libraries. Azure.Provisioning.{ServiceName} contains the resources specific to a particular service.
+
+## Getting started
+
+### Install the package
+
+Install the specific service libraries for .NET with [NuGet](https://www.nuget.org/) required for your application.
+
+```dotnetcli
+dotnet add package Azure.Provisioning.AppConfiguration
+dotnet add package Azure.Provisioning.ApplicationInsights
+dotnet add package Azure.Provisioning.CognitiveServices
+dotnet add package Azure.Provisioning.CosmosDB
+dotnet add package Azure.Provisioning.EventHubs
+dotnet add package Azure.Provisioning.KeyVault
+dotnet add package Azure.Provisioning.OperationalInsights
+dotnet add package Azure.Provisioning.PostgreSql
+dotnet add package Azure.Provisioning.Redis
+dotnet add package Azure.Provisioning.Resources
+dotnet add package Azure.Provisioning.Search
+dotnet add package Azure.Provisioning.ServiceBus
+dotnet add package Azure.Provisioning.SignalR
+dotnet add package Azure.Provisioning.Sql
+dotnet add package Azure.Provisioning.Storage
```

Comment:
Why are we adding these all?  Maybe just do KV since that's all you're showing in the sample.  We can add a comment about all the other `Azure.Provisioning.*` packages.